x-pack/qa/core-rest-tests-with-security/src/yamlRestTest/java/org/elasticsearch/xpack/security/CoreWithSecurityClientYamlTestSuiteIT.java

```diff
@@ -28,6 +29,14 @@ public class CoreWithSecurityClientYamlTestSuiteIT extends ESClientYamlSuiteTest
     private static final String USER = Objects.requireNonNull(System.getProperty("tests.rest.cluster.username", "test_admin"));
     private static final String PASS = Objects.requireNonNull(System.getProperty("tests.rest.cluster.password", "x-pack-test-password"));
 
+    private static final String ROLES = """
+        get_alias_test_role:
+          cluster: [ ]
+          indices:
+            - names: ["test*", "myindex", "non-existent", "another-non-existent", "foo"]
```

Comment:
Granting only necessary privileges for get alias YAML tests. This also includes privileges for missing names in order to avoid changing tests that expect 404 instead of 403.